synapse/rest/media/v1/filepath.py

```diff
@@ -16,11 +17,12 @@
 import functools
 import os
 import re
+from typing import Callable, List
 
 NEW_FORMAT_ID_RE = re.compile(r"^\d\d\d\d-\d\d-\d\d")
 
 
-def _wrap_in_base_path(func):
+def _wrap_in_base_path(func: "Callable[..., str]") -> "Callable[..., str]":
```

Comment:
Why the quotes?

Reply:
I believe that `...` breaks on Python 3.5? We seem to quote similar things in synapse/storage/database.py, but not other places...

Reply:
https://github.com/matrix-org/synapse/blob/1315a2e8be702a513d49c1142e9e52b642286635/synapse/storage/database.py#L177-L178